db/changelog/v-2/01-changeset-test.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <changeSet id="1" author="petertrr" context="dev or prod">
+        <preConditions onFail="MARK_RAN">
+            <tableExists tableName="test"/>
+        </preConditions>
+        <addColumn tableName="test">
+            <column name="plugin_name" type="varchar(250)" value="N/A"/>
+        </addColumn>
+        <rollback>
+            <dropColumn tableName="test" columnName="plugin_name"/>
```

Comment:
_addColumn_ already implicitly has a _rollback_ implementation. You don't need to add this to the _changeSet_.

Reply:
Thanks, removed